examples/main/README.md

```diff
@@ -1,6 +1,6 @@
-# llama.cpp/examples/main
+# llama.cpp/examples/llama-cli
```

Comment:
These titles currently reflect the path of the example within the `llama.cpp` tree, so `llama.cpp/examples/main` is the correct one